### ISSUE TITLE
Document standard(reserved) Keys in volumeReplicationClass parameters

### ISSUE THREAD

**Issue by Madhu-1** [kube-storage/volume-replication-operator#20]
```yaml
storage.openshift.io/replication-secret-name: secret-namespace
storage.openshift.io/replication-secret-namespace: default
```

Document the reserved keys in the volumeReplicationClass parameters. all the keys starting with the `storage.openshift.io` is going to the reserved for the volume-replication-operator. The example above keys can be specified by the admin in VRC in that case the operation will fetch that keys and gets the secrets and passes it to the driver.

**Comment by umangachapagain** [comment]
Should I add this to the main README file in the design section?

**Comment by Madhu-1** [comment]
Yes this need to be added to README

**Comment by Madhu-1** [comment]
keys are defined here

https://github.com/kube-storage/volume-replication-operator/blob/0b6386db19812bd49dd40eb7b297af65ff4bcb1e/controllers/parameters.go#L30-L33

**Comment by umangachapagain** [comment]
Fixed by #39
